content/partials/specs/versions-macos-xcode-12-0.md

```diff
@@ -73,15 +73,14 @@ weight: 5
 - **1.8.0_275** (default) JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home`
 - **1.7.0_285** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-7.jdk/Contents/Home`
 
-## Xcode 12.4 (12D4e)
+## Xcode versions
 
-This is the Xcode version used by default when you select `12.4` in build settings in the workflow
-editor for Flutter apps or set Xcode version to `12.4` in your codemagic.yaml file.
-Other available versions are listed [here](#other-xcode-versions).
-
-Xcode path: `/Applications/Xcode-12.4.app`
+- 12.4 (12D4e) `/Applications/Xcode-12.4.app`
+- 12.3 (12C33) `/Applications/Xcode-12.3.app`
+- 12.2 (12B45b) `/Applications/Xcode-12.2.app`
+- 12.1.1 (12A7605b) `/Applications/Xcode-12.1.1.app`

```

Comment:
maybe we can also add aliases for this one and 12.0.1 below like ", also selected when specifying `12.x` in Xcode version settings" as we did with 13.x ?